### ISSUE TITLE
Add fingerprinting to locale.json files

### ISSUE THREAD

**Issue by mydea** [Cropster/ember-l10n#7]
We want the locale files (e.g. `en.json`) to be properly cached by the browser. However, because ember-cli's fingerprinting doesn't work for this case, we need to implement this manually.

The ideal solution would be:
- When generating a new locale.json file, generate it with a random fingerprint (e.g. `en-12345.json`).
- Automatically save this fingerprint e.g. in the environment.js

E.g.:

``` javascript
// environment.js
var ENV = {
  // ...
  'ember-l10n': { 
    fingerprint: '12345' 
  }
}
```

**Comment by arm1n** [comment]
@mydea Isn't this already fixed by #13? If so, please close this issue - thanks!

**Comment by mydea** [comment]
Yes, I forgot to close this :)
